dashboard/src/components/Hint/Hint.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import { HelpCircle } from "react-feather";
+import * as ReactTooltip from "react-tooltip";
+
+import "./Hint.css";
+
+interface IHintProps {
+  reactTooltipOpts?: any;
+}
+
+export class Hint extends React.Component<IHintProps> {
+  public render() {
+    return (
+      <React.Fragment>
+        <a data-tip={true} data-for="syncJobHelp">
+          <HelpCircle className="icon" color="white" fill="#5F6369" />
+        </a>
+        <ReactTooltip
+          id="syncJobHelp"
```

Comment:
Make this `id` customizable. Identifiers must be unique per page. If we add more than one `Hint` component per page we will have duplicated identifiers